### PR TITLE
typo, grammar nits

### PR DESCRIPTION
**Pull request by brettz9**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Code Smells, or maintainability issues, are raised for places of code which migh
 * Boolean literals should not be redundant ([`no-redundant-boolean`])
 * "switch" statements should have at least 3 "case" clauses ([`no-small-switch`])
 * "catch" clauses should do more than rethrow ([`no-useless-catch`])
-* Local variables should not be declared and then immediately returned or thrown ([`prefer-immediate-return`]) (:wrench: *fixable*)
+* Local variables should not be declared and then be immediately returned or thrown ([`prefer-immediate-return`]) (:wrench: *fixable*)
```
Comment (ganncamp) [inline README.md]: I disagree with this change

File: README.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
 # eslint-plugin-sonarjs [![npm version](https://badge.fury.io/js/eslint-plugin-sonarjs.svg)](https://badge.fury.io/js/eslint-plugin-sonarjs) [![Build Status](https://travis-ci.org/SonarSource/eslint-plugin-sonarjs.svg?branch=master)](https://travis-ci.org/SonarSource/eslint-plugin-sonarjs) [![Quality Gate](https://sonarcloud.io/api/project_badges/measure?project=eslint-plugin-sonarjs&metric=alert_status)](https://sonarcloud.io/dashboard?id=eslint-plugin-sonarjs) [![Coverage](https://sonarcloud.io/api/project_badges/measure?project=eslint-plugin-sonarjs&metric=coverage)](https://sonarcloud.io/dashboard?id=eslint-plugin-sonarjs)
 
-SonarJS rules for ESLint detecting bugs and suspicious patterns in your code.
+SonarJS rules for ESLint to detect bugs and suspicious patterns in your code.
```
Comment (ganncamp) [inline README.md]: IMO this could go either way

File: README.md (hunk 1 above)
Comment (brettz9) [reply README.md]: Yeah, not sure what I was thinking here...

File: README.md (hunk 2 above)
Comment (brettz9) [reply README.md]: It sounds very awkward to me, but I can't place a specific grammar rule, so your call...

Comment (brettz9) [reply README.md]: I think it is because it is ambiguous, like "ESLint detecting bugs" sounds like one noun.

**Comment by vilchik-elena** [comment]
@brettz9 thanks!

**Comment by brettz9** [comment]
Looks like some quite useful rules here, thanks! Hopeful of integrating a good number of them to [my config](https://github.com/brettz9/eslint-config-ash-nazg)

**Pull request merged**